### PR TITLE
some fixes in Fixture

### PR DESCRIPTION
**Pull request by sebastianbaginski**
Fixes https://github.com/magic-lang/ooc-kean/issues/1265
If someone has better idea how to fix this then please let me know
@marcusnaslund

### CODE REVIEW

**Comment by marcusnaslund** [comment]
Not too terrible, but it's a shame if we can't trust `match` statements.

**Comment by marcusnaslund** [comment]
Did you try to match on `this val class`, remove `value`, and instead do `this val toString()` on every line?

**Comment by sebastianbaginski** [comment]
Yes, `this val class` crashed at runtime. `T class` caused compilation error.

**Comment by marcusnaslund** [comment]
Alright then. Have you tried this for failing comparisons of `String`?

**Comment by sebastianbaginski** [comment]
I didn't, only for text and some numeric types

**Comment by marcusnaslund** [comment]
It's alright, it wasn't good before and it'll be replaced with `Text` soon.

**Comment by marcusnaslund** [comment]
Wait, there is no `String` clause amongst these `if` statements. That explains a lot. Would you add that too?

**Comment by sebastianbaginski** [comment]
ok

**Comment by marcusnaslund** [comment]
Makes very little sense to have `toString` if it cannot handle `String`. :)

**Comment by sebastianbaginski** [comment]
Updated.

File: source/sdk/lang/types.ooc [hunk 1]
```diff
@@ -128,21 +128,36 @@ Cell: class <T> {
...
-			case value: ULong => value toString()
-			case value: Float => "%.8f" formatFloat(value)
-			case value: Double => "%.12f" formatDouble(value)
-			case value: LDouble => "%.12f" formatLDouble(value)
-			case => raise("[Cell] toString() is not implemented on the specified type"); null
-		}
+		result: String
+		if (T inheritsFrom?(Text))
+			result = (this val as Text) toString()
+		if (T inheritsFrom?(String))
```
Comment (marcusnaslund) [inline source/sdk/lang/types.ooc]: missing `else`

Comment (sebastianbaginski) [inline source/sdk/lang/types.ooc]: ...

Comment (sebastianbaginski) [inline source/sdk/lang/types.ooc]: fixed

**Comment by marcusnaslund** [comment]
Works well now, nice.

**Pull request merged**